recipes/rangesnext/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class RangesnextConan(ConanFile):
+    name = "rangesnext"
+    description = "ranges features for C++23 ported to C++20"
+    topics = ("conan", "rangesnext", "ranges", "backport", "backport-cpp")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/cor3ntin/rangesnext"
+    license = "BSL-1.0"
+    exports_sources = "patches/**"
+    settings = "compiler"
+
+    _compilers_minimum_version = {
+        "gcc": "10",
+        "Visual Studio": "19",
+        "clang": "13"
```

Comment:
```suggestion
        "clang": "13",
        "apple-clang": "11"
```

https://en.cppreference.com/w/cpp/compiler_support

Reply:
I removed apple-clang explicitly. It doesn't support C++20 `<ranges>` yet, and it returns this error:

```
/Users/cesar/.conan/data/rangesnext/cci.20210426/user/testing/package/0395d3cb61e1529b57a221cb5d56a0edf01dc16c/include/cor3ntin/rangesnext/__detail.hpp:10:10: fatal error: 'ranges' file
      not found
#include <ranges>
         ^~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/test_package.dir/test_package.cpp.o] Error 1
make[1]: *** [CMakeFiles/test_package.dir/all] Error 2
make: *** [all] Error 2
```